src/vellum/workflows/events/stream.py

```diff
@@ -26,3 +30,65 @@ def __iter__(self) -> Iterator[EventType]:
     def __next__(self) -> EventType:
         """Get the next event from the underlying generator."""
         return next(self._event_generator)
+
+    @staticmethod
+    def stream_initialization_exception(
+        exception: "WorkflowInitializationException",
+    ) -> "WorkflowEventGenerator[WorkflowEvent]":
+        """
+        Stream a workflow initiated event followed by a workflow rejected event for an initialization exception.
+
+        Args:
+            exception: The WorkflowInitializationException to stream events for
+
+        Returns:
+            WorkflowEventGenerator yielding initiated and rejected events
+        """
+        from vellum.workflows.context import get_execution_context
+        from vellum.workflows.events.workflow import (
+            WorkflowEvent,
+            WorkflowExecutionInitiatedBody,
+            WorkflowExecutionInitiatedEvent,
+            WorkflowExecutionRejectedBody,
+            WorkflowExecutionRejectedEvent,
+        )
+        from vellum.workflows.inputs import BaseInputs
+        from vellum.workflows.nodes.bases.base import BaseNode
+        from vellum.workflows.state import BaseState
+        from vellum.workflows.workflows.base import BaseWorkflow
+
+        class _PlaceholderNode(BaseNode):
+            def run(self):
+                return self.Outputs()
```

Comment:
delete this placeholder stuff. Instead, `workflow_definition=exception.definition`

Reply:
Update the `WorkflowInitializationException` init method to accept a workflow definition kwarg that should be accessible from all consumers